web/layout_attributes/using_flex/my_element.dart

```diff
@@ -0,0 +1,13 @@
+@HtmlImport('my_element.html')
+library my_element;
+
+import 'package:web_components/web_components.dart' show HtmlImport;
+import 'package:polymer/polymer.dart';
+import 'package:polymer_elements/paper_styles.dart';
+
+/// Silence analyzer [JsArray]
+@jsProxyReflectable
```

Comment:
tldr; You don't need to put it on anything, just add the `JsProxy` mixin class (which is included already on `PolymerElement`) and you are good to go.

The full story here is the reflectable package now has a capability called `subtypeQuantifyCapability`, which essentially makes it so if you put a reflectable annotation on a class then any class which implements, extends, or mixes in that class will also be reflectable. So the JsProxy class itself now has @jsProxyReflectable and thus anything that mixes it in also is reflectable, its pretty neat :).